cli/config/configuration.toml.in

```diff
@@ -8,6 +8,7 @@
 [hypervisor.qemu]
 path = "@QEMUPATH@"
 kernel = "@KERNELPATH@"
+initrd = "@INITRDPATH@"
```

Comment:
Could you update `kata-env.go` to add a `[Initrd]` entry for parity with `[Kernel]` and `[Image]` (and bump `formatVersion` to `1.0.10`)?